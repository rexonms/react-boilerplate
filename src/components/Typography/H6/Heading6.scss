.default {
  font-family: $fontDefault;
  font-size: $fontSizeSmall;
  color: $fontColorDark;
  margin: 0;
  padding: 0;
}