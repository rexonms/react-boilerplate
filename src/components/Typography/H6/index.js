import React from 'react';
import PropTypes from 'prop-types';

import styles from './Heading6.scss';

const Heading6 = props => (
  <h6 className={styles.default}>{props.labelText}</h6>
);

Heading6.defaultProps = {};

Heading6.propTypes = {
  /** Text for the component */
  labelText: PropTypes.string.isRequired,


};

export default Heading6;
