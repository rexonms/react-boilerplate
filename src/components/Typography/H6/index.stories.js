import React from 'react';
import { storiesOf } from '@storybook/react';
import { withKnobs, select } from '@storybook/addon-knobs/react';
import { withInfo } from '@storybook/addon-info';

import Heading6 from './index';

const stories = storiesOf('Components/Typography/H6', module);
stories.addDecorator(withKnobs);

// Knobs as dynamic variables.
stories.addDecorator(story => (story()));

stories.add('default', withInfo(`
  Heading 
  ~~~jsu
    <Heading6 labelText='Heading 6' />);
  ~~~
`)(() => {
  return (
    <Heading6 labelText='Heading 6' />);
}));

