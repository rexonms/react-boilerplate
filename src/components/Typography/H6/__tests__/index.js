import React from 'react';
import renderer from 'react-test-renderer';

import H6 from './../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <H6 labelText='heading'/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
