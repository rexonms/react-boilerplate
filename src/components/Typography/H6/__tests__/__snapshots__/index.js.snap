// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h6
  className="default"
>
  heading
</h6>
`;
